src/main/java/de/unistuttgart/informatik/fius/icge/simulation/Simulation.java

```diff
@@ -435,4 +434,20 @@ private synchronized void tick(Runnable afterwards) {
             super(sim);
         }
     }
+
+    /**
+     * Add an enity to the list
+     * @param entity
+     */
+    public void addEntity (Entity entity) {
+        this._entities.add(entity);
+    }
+
+    /**
+     * Return entities
+     */
+    public ArrayList<Entity> getEntities() {
+        return this._entities;
+    }
+

```

Comment:
The `Simulation` class does already have the `entities()` method which returns all the `Entity`s which are alive in that `Simulation`.

Reply:
(This also apparently doesn't have anything to do with the speed slider)